docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:18.04
+MAINTAINER Bo Zhang (zhangb1@email.chop.edu)
+
+RUN apt-get update -y && \
+apt-get install -y python3.6 python3-pip wget git bedtools pandoc && \
+apt-get install -y build-essential libcairo2-dev libxt-dev libcurl4-gnutls-dev libxml2-dev libssl-dev libmariadbclient-dev libfontconfig1-dev libmagick++-dev libharfbuzz-dev libfribidi-dev libpq-dev
+
+#install R 3.6
+
+ENV TZ=America/New_York
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
why do we need this timezone setup?

Reply:
this need to be setup, because when install R 3.6, the software will ask you the timezone info. Or you have to manually enter the timezone info.